build.py

```diff
@@ -323,7 +332,12 @@ def fmt():
 
 
 def main():
-    # Create the top level command line parser:
+    """
+    Create the top level command line parser:
```

Comment:
This is not what the method does. If `pylint` really want this comment then write something that describes the method, not just the first line.